client/dashboard/sites/settings/index.tsx

```diff
@@ -20,15 +21,18 @@ export default function SiteSettings() {
 		return null;
 	}
 
+	const { site } = siteData;
+
 	return (
 		<PageLayout size="small">
 			<PageHeader title={ __( 'Settings' ) } />
-			<Heading>{ __( 'General' ) }</Heading>
+			<Heading level={ 2 }>{ __( 'General' ) }</Heading>
```

Comment:
`2` is the default level.

Reply:
Oh nice, so `PageHeader` is `h1` and the next immediate level is `Heading` 😄 I just wanted to be explicit. But removed!